src/test/java/api/support/fixtures/ItemsFixture.java

```diff
@@ -351,7 +351,9 @@ public HoldingBuilder thirdFloorHoldings()
     return new HoldingBuilder()
       .withPermanentLocation(locationsFixture.thirdFloor())
       .withNoTemporaryLocation()
-      .withCallNumber("123456");
+      .withCallNumber("123456")
+      .withCallNumberSuffix("CIRC")
+      .withCallNumberPrefix("PR");
```

Comment:
Why are defaults needed for these properties?

Reply:
they are not required, just for testing purpose.